aclk/agent_cloud_link.c

```diff
@@ -692,8 +693,10 @@ void aclk_add_collector(const char *hostname, const char *plugin_name, const cha
 
     if (unlikely(agent_state == AGENT_INITIALIZING))
         last_init_sequence = now_realtime_sec();
-    else
-        aclk_queue_query("connector", NULL, NULL, NULL, 0, 1, ACLK_CMD_ONCONNECT);
+    else {
+        if (unlikely(aclk_queue_query("connector", NULL, NULL, NULL, 0, 1, ACLK_CMD_ONCONNECT)))
```

Comment:
This seems a bit opaque - why has the command changed in some places?

Reply:
Just added error check -- debug logging